sdk/tables/azure-data-tables/azure/data/tables/__init__.py

```diff
@@ -6,6 +6,7 @@
 from azure.data.tables._models import TableServiceStats
 
 from ._entity import TableEntity, EntityProperty, EdmType
+from ._error import RequestEntityTooLargeError
```

Comment:
I would prefer `RequestTooLargeError`
`Entity` already means something different in the Tables context :)